Tests/ResilientDecodingTests/ResilientRawRepresentableDictionaryTests.swift

```diff
@@ -34,6 +34,9 @@ final class ResilientRawRepresentableDictionaryTests: XCTestCase {
     #endif
   }
 
+  /**
+   - note: We keep the non-optional properties in the JSON so they do not report errors
+   */
```

Comment:
Since this test name claims no errors should we add `expectedErrorCount: 0` to our decoding method?